care/facility/tests/test_patient_api.py

```diff
@@ -111,6 +111,10 @@ def test_patient_notes(self):
             data.keys(), [item.value for item in ExpectedPatientNoteKeys]
         )
 
+        created_by_local_user_content = data["created_by_local_user"]
+
+        self.assertIsInstance(created_by_local_user_content, bool)
```

Comment:
You need to check whether the value is expected instead of its type

and create another user from a different facility and check if that returns false